charts/quantum-serverless/charts/gateway/values.yaml

```diff
@@ -8,6 +8,7 @@ useCertManager: false
 application:
 #  command: [ "gunicorn", "gateway.wsgi:application", "--bind", "0.0.0.0:8000", "--workers=3" ]
   debug: 0
+  secretKey: "django-insecure-&)i3b5aue*#-i6k9i-03qm(d!0h&662lbhj12on_*gimn3x8p7"
```

Comment:
If my previous comment apply we could remove this line letting only the default value in `gateway`:

https://github.com/Qiskit-Extensions/quantum-serverless/pull/905/files#diff-d3fab2ab912c6eb5c879651a1476f64f0551b747bf618b4ab13ddb4602dd45c1R26